contrib/diurnal/example-diurnal-controller.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: diurnalcontroller
+  name: diurnalcontroller
+spec:
+  replicas: 1
+  selector:
+    name: diurnalcontroller
+  template:
+    metadata:
+      labels:
+        name: diurnalcontroller
+    spec:
+      containers:
+        - command: ["diurnal_math", "-ns", "default", "-labels", "name=redis-slave", "-sample", "300", "-countfn", "10+10*sin(t)"]
```

Comment:
The actual controller takes in a list of times and replica counts. diurnal_math is a wrapper which instead lets you provide a formula and sample rate at the request of @thockin. Perhaps I should add another example using that.

At the moment the user also provides some maximum rate (the example currently uses the default value, I'll make it explicit) of count change per minute and the controller will transition from one value to another as fast as possible at that rate. It might be easier to just specify some step size instead.